stat11_backend/stat11/views/team.py

```diff
@@ -22,4 +22,10 @@ def get_serializer_class(self):
     def all_teams_and_batters(self, request):
         all_list = self.get_queryset()
         res = get_team_batter_data(all_list[0].id)
+        return Response(res)
+    
+    @action(detail=False, methods=['get'])
+    def all_teams_and_bowlers(self, request):
+        all_list = self.get_queryset()
+        res = get_team_bowler_data(all_list[0].id)
         return Response(res)
```

Comment:
Again incomplete functions.....why are you looking for only the first query objects?

Reply:
Resolved